geode-cq/src/upgradeTest/java/org/apache/geode/internal/cache/tier/sockets/ClientServerMiscBCDUnitTest.java

```diff
@@ -226,8 +228,10 @@ public void giiEventQueueFromOldToCurrentMemberShouldSucceed() {
 
   @Test
   public void giiEventQueueFromCurrentToOldMemberShouldSucceed() {
+    final IgnoredException expectedEx =
```

Comment:
No need to invoke getName() on the class -- you can just use .class directly. Also, I recommend using the try-resource syntax since IgnoredException is AutoCloseable:
```
try (IgnoredException ie = addIgnoredException(ConnectException.class)) {
  giiEventQueueShouldSucceedWithMixedVersions(VersionManager.CURRENT_VERSION, testVersion);
}

Reply:
Also, DUnit automatically removes IgnoredExceptions after the test, so another alternative is:
```
addIgnoredException(ConnectException.class);
giiEventQueueShouldSucceedWithMixedVersions(VersionManager.CURRENT_VERSION, testVersion);
```